x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/job_map.tsx

```diff
@@ -172,12 +173,16 @@ export const JobMap: FC<Props> = ({ analyticsId, modelId }) => {
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
-      <div style={{ height: height - parseInt(theme.gutterTypes.gutterLarge, 10) - 20 }} ref={ref}>
+      <div
+        style={{ height: height - parseInt(euiTheme.gutterTypes.gutterLarge, 10) - 20 }}
```

Comment:
We should avoid using magic numbers, so I presume `20` is the padding value from `getLayoutOptions`. 
Also, have you tried using a vertical flex layout to stretch this element instead of computing the height manually?

Reply:
Thanks for taking a look, @darnautov  😄 I have not tried using vertical flex layout. A good idea but I think outside the scope of this PR - happy to explore in a follow up as a bug fix after dark theme goes in if that sounds alright on your end.